.github/workflows/android.yml

```diff
@@ -48,7 +48,10 @@ jobs:
           api-level: 24
           arch: x86_64
           disable-animations: true
-          script: ./gradlew build connectedCheck jacocoTestReport
+          script: |
```

Comment:
I have been following your discussion thread on the forum. Am I right that there is still one problem with the Mock?

Reply:
No, I managed to solve it, and I posted those lines of code if anyone needs it. What it does is that it first installs the app on the emulator, then grants permission for mocking the location, and finally launches the tests.